fugue-deploy-aws/src/main/java/com/symphony/oss/fugue/aws/deploy/AwsFugueDeploy.java

```diff
@@ -1002,6 +1006,8 @@ private String createRole(Name name, String trustDocument, String ...policyArnLi
         log_.warn("Interrupted", e1);
       }
       log_.debug("Created role " + roleName + ", waiting to allow role to become active...OK carry on.");
```

Comment:
this message does not make sense

change to "will wait to allow role to become active before launching init task"